Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -4131,6 +4131,123 @@ def get_cve_command(args: dict) -> list[CommandResults]:
     return command_results_list
 
 
+# TODO find out where to put this
+CPU_UTIL_INT_TO_HR = {
+    1: 'Lowest',
+    2: 'Low',
+    3: 'Medium',
+    4: 'High',
+    5: 'Highest',
+}
+
+
+CPU_UTIL_HR_TO_INT = {value: key for key, value in CPU_UTIL_INT_TO_HR.items()}
+
+
+def ODS_query_scans_request(**query_params) -> dict:
+    remove_nulls_from_dictionary(query_params)
+    # Temporary solution:
+    return http_request('GET', f'/ods/queries/scans/v1?{"&".join(f"{k}={v}" for k,v in query_params.items())}')
+    # return http_request('GET', '/ods/queries/scans/v1', params=query_params)
+    
+
+def ODS_get_scans_by_id_request(ids: list[str]) -> dict:
+    if not ids:
+        raise DemistoException('No IDs to check.') # TODO  check if better response needed
```

Comment:
arent we doing this if not ids: check twice? Lets make this check only in 1 place

Reply:
done